src/core/ActionMenu/ActionMenu.md

```diff
@@ -1,142 +1,219 @@
-### Default ActionMenu
+`<ActionMenu>` is used for selecting an action if the amount of available actions is so great regular `<Button>` elements would clutter the UI
```

Comment:
```suggestion
`<ActionMenu>` is used for selecting an action if the amount of available actions is so great that regular `<Button>` elements would clutter the UI
```

Reply:
I belive the 'that' word is not needed in this case according to English grammar 😅

Reply:
It's not required, but would improve the readability of the sentence. But you could perhaps simplify by saing "`<ActionMenu>` is used for selecting an action when using regular `<Button>` elements would clutter the UI."

Reply:
I'll accept all versions. Daniel can resolve this :)